pkg/models/kubectl/kubectl.go

```diff
@@ -146,7 +140,7 @@ func (o *operator) CreateKubectlDeploy(username string) error {
 				Spec: v1.PodSpec{
 					Containers: []v1.Container{
 						{Name: "kubectl",
-							Image: DefaultImage,
+							Image: o.kubectlImage,
```

Comment:
It's not a bad idea to have a default image, because If user forget to set kubectl image, we still have a backup option.